lab_1_classify_by_unigrams/main.py

```diff
@@ -20,9 +22,15 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    lowered_text = text.lower()
+    tokens = []
+    for elem in lowered_text:
+        if elem.isalpha() and elem != "º":
```

Comment:
why "º"?

Reply:
because .isalpha thinks that this dergee symbol is alpha. I do not know why, unfortunatly